Makefile

```diff
@@ -21,6 +21,7 @@ sync-push: sync-image
 
 verify:
 	./hack/validate-generated.sh
+	./hack/validate-deps.sh
```

Comment:
We will probably need to use a different test base image than `openshift/release:golang-1.10` in our tests since it does not include glide I think.

Reply:
Im started working on assumption that we would maintain our own base image based on `centos7` where we will have `az, go, glide, etc` as our stack is quite unique.